dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -252,6 +252,114 @@ html, body {
   }
 }
 
+/* Styling that keeps the small footer consistent with other page elements */
+$codeApp-left-margin: 25px;
+$small-footer-horizontal-margin: 15px;
+$small-footer-standard-width: 400px;
+
+/* Shared styles so flyout looks like footer */
+.small-footer, #copyright-flyout, #more-menu {
+  background-color: #ddd; /* Matches blockly flyout */
```

Comment:
This might be premature, one option could be fitting this in `shared/css/color.scss` as e.g. `$blockly-flyout-color: #ddd;`. Thought only benefit unless we use the color again / change flyout color would be removing comment necessity